.kit/context/workflows/TASK-COMPLETION-PROTOCOL.md

```
@@ -22,15 +22,15 @@
 4. ✅ **No regressions introduced** (overall pass rate maintained or improved)
 5. ✅ **Documentation updated** (README, CHANGELOG, ADRs if applicable)
 6. ✅ **CI/CD passing** (GitHub Actions green)
-7. ✅ **Handoff document created** (`delegation/handoffs/HANDOFF-TO-*.md`)
+7. ✅ **Handoff document created** (`.kit/delegation/handoffs/HANDOFF-TO-*.md`)
 
 ---
 
 ## Handoff Document Format
 
 ### Filename:
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)